core/test/test_move_to.cpp

```diff
@@ -156,6 +157,34 @@ TEST_F(PandaMoveTo, poseIKFrameAttachedSubframeTarget) {
 	EXPECT_ONE_SOLUTION;
 }
 
+// This test require a running rosmaster
+TEST(Task, taskMoveConstructor) {
+	auto create_task = [] {
+		moveit::core::RobotModelConstPtr robot_model = getModel();
+		Task t("first");
+		t.setRobotModel(robot_model);
+		auto ref = new stages::FixedState("fixed");
+		auto scene = std::make_shared<planning_scene::PlanningScene>(t.getRobotModel());
+		ref->setState(scene);
+
+		t.add(Stage::pointer(ref));
+		t.add(std::make_unique<ConnectMockup>());
+		t.add(std::make_unique<MonitoringGeneratorMockup>(ref));
+		return t;
+	};
+
+	// Segfaults when introspection is enabled
```

Comment:
```suggestion
	// Does not segfault when introspection is enabled
```

Reply:
Yeah, I'll update the test description to mention that issue it's fixing